src/query/api/v1/httpd/handler.go

```diff
@@ -120,7 +122,14 @@ func (h *Handler) RegisterRoutes() error {
 
 	h.Router.HandleFunc(remote.PromReadURL, logged(promRemoteReadHandler).ServeHTTP).Methods(remote.PromReadHTTPMethod)
 	h.Router.HandleFunc(remote.PromWriteURL, logged(promRemoteWriteHandler).ServeHTTP).Methods(remote.PromWriteHTTPMethod)
-	h.Router.HandleFunc(native.PromReadURL, logged(native.NewPromReadHandler(h.engine)).ServeHTTP).Methods(native.PromReadHTTPMethod)
+
+	// Prometheus native query endpoints, registered under both the default
+	// Prometheus server route and the Prometheus native read endpoint route
+	promNativeRead := logged(native.NewPromReadHandler(h.engine)).ServeHTTP
+	h.Router.HandleFunc(prometheus.DefaultQueryRangeURL, promNativeRead).Methods(prometheus.DefaultQueryRangeMethod)
```

Comment:
does it make sense to have both or shall we just update native.PromReadURL to point to /api/v1/query_range ?